src/server/server.ts

```diff
@@ -216,6 +227,33 @@ export default class Server implements Hub.RouteBuilder {
     }
   }
 
+  private async asyncProcess(req: express.Request, res: express.Response) {
+      return new Promise<void>((resolve, reject) => {
+          const child = spawn.fork(`./src/actions/actions_process.ts`)
+          child.on("message", (actionResponse) => {
+              // Some versions of Looker do not look at the "success" value in the response
+              // if the action returns a 200 status code, even though the Action API specs otherwise.
+              // So we force a non-200 status code as a workaround.
+              winston.info("Success? " + actionResponse.success + " || and what: " + JSON.stringify(actionResponse))
+              if (!actionResponse.success) {
+                  res.status(400)
+              }
+              res.json(actionResponse)
+
+              child.kill()
+              resolve()
+          }).on("error", (err) => { winston.warn(err.message); reject(err) })
+          const data = {
+              body: req.body,
+              actionId: req.params.actionId,
+              instanceId: req.header("x-looker-instance"),
+              webhookId: req.header("x-looker-webhook-id"),
+              userAgent: req.header("user-agent"),
+          }
+          child.send(data)
```

Comment:
the child process imports this as `any`. can we define an explicit interface here that can be shared between the two files.

also, i think it would be prudent to genericize this interface a bit to make it easy to move other jobs into child processes (like the form request, etc.)